test/unit/spec/common/cellComponents/tabs/jobActionDropdown-Spec.js

```diff
@@ -53,89 +55,206 @@ define([
     });
 
     describe('The job action dropdown instance', () => {
-        let container;
-        beforeEach(function () {
+        beforeEach(() => {
             container = document.createElement('div');
-            this.jobActionDropdownInstance = createInstance();
+            jobActionDropdownInstance = createInstance();
```

Comment:
switched to using a variable instead of `this.jobActionDropdownInstance` throughout.